File: Codeforces rounds/Codeforces rounds/Codeforces Round 143 (Div. 2)/A.cpp
// https://codeforces.com/contest/231/problem/A

#include <bits/stdc++.h>

using namespace std;

typedef unsigned int uint;

int main(void) {
  ios_base::sync_with_stdio(false);
  cin.tie(NULL);

  uint n;
  cin >> n;

  uint ans = 0;

  for (uint i = 0; i < n; i++) {
    uint a, b, c;
    cin >> a >> b >> c;

    if (a + b + c >= 2) {
      ans++;
    }
  }

  cout << ans << '\n';
}

File: Codeforces rounds/Codeforces rounds/Codeforces Round 143 (Div. 2)/A.py
# https://codeforces.com/contest/231/problem/A


def main():

    n: int = int(input())

    ans: int = 0

    for _ in range(n):
        a, b, c = map(int, input().split())
        if a + b + c >= 2:
            ans += 1

    print(ans)


if __name__ == '__main__':
    main()
